pymatgen/io/vasp/sets.py

```diff
@@ -362,7 +365,7 @@ def write_input(
             files_to_transfer=self.files_to_transfer,
         )
 
-    def as_dict(self, verbosity=2):
+    def as_dict(self, verbosity: int = 2) -> dict:  # TODO: redefine verbosity?
```

Comment:
`verbosity` doesn't really reflect what is does, perhaps deprecate it and rename to something like `exclude_structure`?

Reply:
`verbosity`, while misleading in many cases, is the canonical control parameter name in `monty` and so changing it would likely encounter push back

Reply:
I just notice this when going through `core.sites`, looks like it's intended to tweak the verbosity of dict representations.

But the verbosity as an `int` still is a bit ambiguous to me...https://github.com/materialsproject/pymatgen/blob/5c7889c65289352e053b99bab8ec4d17686abfa6/pymatgen/core/sites.py#L579-L585